nav2-usability/scenario-code/dynamic-linking/dynamic_linking_vrouter.dart

```diff
@@ -0,0 +1,159 @@
+// Copyright 2021, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Dynamic linking example
+/// Done using VRouter
+
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:vrouter/vrouter.dart';
+
+void main() {
+  runApp(WishlistApp());
+}
+
+class Wishlist {
+  final String id;
+
+  Wishlist(this.id);
+}
+
+class AppState extends ChangeNotifier {
+  final List<Wishlist> wishlists = <Wishlist>[];
+
+  void addWishlist(Wishlist wishlist) {
+    wishlists.add(wishlist);
+    notifyListeners();
+  }
+}
+
+class WishlistApp extends StatefulWidget {
+  @override
+  _WishlistAppState createState() => _WishlistAppState();
+}
+
+class _WishlistAppState extends State<WishlistApp> {
+  final AppState _appState = AppState();
+  final vRouterKey;
+
+  _WishlistAppState() : vRouterKey = GlobalKey<VRouterState>() {
+    _appState.addListener(() {
+      setState(() {});
+    });
+  }
+
+  @override
+  void dispose() {
+    _appState.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
nit: build should be the last method in the class

Reply:
Nice to know, done in 333401982bbb53e7e6fef16e1cbdbb9306ba73b6.